mongospecs/mixins/crud.py

```
@@ -0,0 +1,323 @@
+class CrudMixin(QueryMixin):
+    # Operations
+    def insert(self, **insert_one_kwargs: t.Any) -> None:
+        """Insert this document"""
+        # Send insert signal
+        signal("insert").send(self.__class__, specs=[self])
+
+        document_dict = self.to_dict()
+        if not self._id:
+            document_dict.pop("_id", None)
+        # Prepare the document to be inserted
+        document = to_refs(document_dict)
```

Ruff (0.8.2)

Undefined name `to_refs`

(F821)